### PR TITLE
Tapping a repository takes you to the page

### PR DESCRIPTION
**Pull request by mattprecious**

### CODE REVIEW

File: src/main/java/com/jakewharton/u2020/ui/trending/TrendingView.java [hunk 1]
```diff
@@ -75,8 +79,8 @@ public TrendingView(Context context, AttributeSet attrs) {
     swipeRefreshView.setColorSchemeResources(R.color.accent);
     swipeRefreshView.setOnRefreshListener(this);
 
-    trendingAdapter.registerAdapterDataObserver(new RecyclerView.AdapterDataObserver() {
-      @Override public void onChanged() {
+        trendingAdapter.registerAdapterDataObserver(new RecyclerView.AdapterDataObserver() {
```
Comment (JakeWharton) [inline src/main/java/com/jakewharton/u2020/ui/trending/TrendingView.java]: Unindnet

File: src/main/java/com/jakewharton/u2020/util/Intents.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
+package com.jakewharton.u2020.util;
```
Comment (JakeWharton) [inline src/main/java/com/jakewharton/u2020/util/Intents.java]: Merge with `misc` package?

**Comment by JakeWharton** [comment]
lgtm

File: src/main/java/com/jakewharton/u2020/util/Intents.java (hunk 2 above)
Comment (mattprecious) [inline src/main/java/com/jakewharton/u2020/util/Intents.java]: The `misc` package is under `ui` and is miscellaneous views/widgets/ui-related things, where as `util` is (mostly) static helpers. I think the separation makes sense.

**Pull request merged**